src/skip-to-content/skip-to-content.scss

```diff
@@ -15,8 +15,9 @@
   position: fixed;
   text-align: center;
   left: 50%;
-  opacity: 0;
   top: $iui-baseline * 2;
+  opacity: 0;
+  z-index: 99;
```

Comment:
I think this should be fine as is, but I want to share with you something that I do every time I use `z-index`:  I do a repo wide search for `z-index` and then think through "ok, what do I want this to be above & below?
![Screen Shot 2022-04-06 at 9 38 54 AM](https://user-images.githubusercontent.com/849817/161987796-d146ebe1-5657-4db9-9770-564048a401f7.png)
So for example, we'd want this be beneath the modal but above most of the rest of the stuff.  So 99 should be fine.
"

Reply:
Good tip, that's what I did! 😄